tests/__init__.py

```diff
@@ -367,6 +367,17 @@ def start_impl_portal(self, params=None, portal=None):
 
```

Comment:
Commit message nit: should be in the form of a description of the change, e.g. `remote-desktop: Add clipboard tests`